src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Scripts/Script+CBXMLHandler.m

```diff
@@ -185,6 +185,9 @@ + (NSString*)brickClassNameForBrickTypeName:(NSString*)brickTypeName
        ([brickTypeName isEqualToString:@"LedOffBrick"])){
         return @"FlashBrick";
     }
+    if (([brickTypeName isEqualToString:@"PenBrick"])){
```

Comment:
I think this can be removed once you added a specific Objective C name to the Brick (`@objc(PenBrick)`).